docs/_temp/Installation.md

```diff
@@ -0,0 +1,146 @@
+In the real world, a project is something being built and/or researched to solve some problem or to open new grounds.
+A DevLake project is a grouping of `pull requests`, `deployments`, or `incidents`. It can be seen as a real-world project or product line. DevLake measures DORA metrics for each project.
+
+![](project_pipeline.png)
+
+_Note: It does not matter if a team works on a particular repository more than another.
+The metrics are calculated over the entire set of repos, and the values are accumulated.
+More on that: [Debugging DORA Issue Metrics](docs/Troubleshooting/Dashboard.md#debugging-dora-issue-metrics)_
+
+## 3. As a team lead, how many DevLake projects do I need?
+
+Because of its simplicity, the concept is flexible: you decide how to arrange `pull requests`, `deployments`, and `incidents`
+either by your specific projects, by teams, technology, or any other way.
+
+The examples below show the patterns of how to organize your projects.
+To keep things simple we assume that we work with **GitHub** repos, **Jira** boards, 
+and **Jenkins CI/CD** deploys in each GitHub repo.
+
+The same would apply to other repos (e.g. GitLab or BitBucket), boards (e.g. TAPD), 
+or CI/CD (e.g. GitLab CI, Azure DevOps).
+
+## 4. Building use case 1
+
+There are `2 teams` with `2 boards`, 3 `repos`, and 3 `cicd pipelines`. 
+One of the repos is shared between both teams.
+
+![](project_use_case_1.png)
+
+Let's build this example.
+
+### 4.1. Organizing projects
+DORA is good for testing effects of the new changes in team's methodology. 
+So, naturally, we will create 2 `projects`, one for each team in this case.
+
+![](create_project_1.png)
+![](create_project_2.png)
+
+### 4.2. Creating connections
+
+For GitHub `repos` we will create:
+- 1 connection for Team A
+- 1 connection for Team B
+- 1 connection for shared repository
+
+TODO: or just 1 connection per team with all necessary repos?
+
+So we can simply combine work in the `shared repo` with each of the teams.
+The connections to retrieve the `deployments` of Jenkins will be arranged the same way.
+
+For JIRA `incident boards` we will create 1 connection per each board.
+
+#### 4.2.1 GitHub connection
+1. Create a GitHub access token specified in the [official GitHub Guide](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)
+2. Navigate to the "Connections" view
+![](configure_connection_github_1.png)
+3. Click "Create New Connection" and fill the form similar as specified in the following screenshots.
+![](configure_connection_github_2.png)
+![](configure_connection_github_3.png)
+4. ⚠️ Warning: Make sure the token has the fields repo:status, repo_deployment, read:user, read:org. 
+   - If collecting private repositories, also include the repo field.
+5. Click "Test Connection". If no error appears, click "Save Connection".
+
+#### 4.2.2 GitHub connection scope
+TODO
+
+#### JIRA
```

Comment:
With the example of creating a GitHub connection above, we don't have to add the diagrams for creating connections for other data sources, but attach the existing URLs here. 

The URLs can be found in this folder https://devlake.apache.org/docs/Configuration, please use the relative URL in the doc.